security/wildfly/pom.xml

```diff
@@ -121,6 +125,14 @@
                                             <version>${feature.pack.version}</version>
                                         </feature-pack>
                                     </feature-packs>
+                                    <channels>
+                                        <channel>
+                                            <manifest>
+                                                <groupId>org.wildfly.channels</groupId>
+                                                <artifactId>wildfly-ee</artifactId>
+                                            </manifest>
+                                        </channel>
+                                    </channels>
```

Comment:
Is this added as an aid to project like RESTEasy?

Any concerns about only including wildfly-ee?

Reply:
This was also to always use the latest release. It's not needed and I can remove it. 

The reason for wildfly-ee was that is just what we were using before. IoW since the feature-pack is wildfly-ee-galleon-pack, we need to use the wildfly-ee channel.